.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+	"java.configuration.updateBuildConfiguration": "interactive",
```

Comment:
Could the vscode config be gitingored?

Reply:
Yeah, I can remove this file

Reply:
removed, and the folder is now gitignored